lib/pages/chat/input_bar.dart

```diff
@@ -401,6 +403,9 @@ class InputBar extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final useShortCuts = (AppConfig.sendOnEnter ?? !PlatformInfos.isMobile);
+    // #Pangea
+    final bool maxLength = controller?.text.length == 1000;
```

Comment:
Hello! Can you move this out of the build function and make it a getter function? This would look like:
```
bool get isMaxLength => controller?.text.length == 1000;
```

That way it doesn't have to be reassigned on each build.
Also, you might be able to move this getter function to the PangeaTextController class - I think that functionality more belongs there. That way the function in the text controller would look like:
```
bool get isMaxLength => text.length == 1000;
```
and reference it in the view code like:
```
controller?.isMaxLength
```